### PR TITLE
Add Zero data experience

### PR DESCRIPTION
**Pull request by beccamc**
Zero data for Areas and Objectives 
Areas:
image

Objectives:
image

- Also changed the initialization of areas and objectives to only occur from the OkrMain page.

### CODE REVIEW

File: src/DetailView/DetailOKRList.tsx [hunk 1]
```diff
@@ -16,22 +17,28 @@ export class DetailOKRList extends React.Component<IDetailOKRListProps, {}> {
...
+        else {
+            const areaName = selectedArea ? selectedArea.Name : "";
+            return <ObjectiveZeroData areaName={areaName} />
+        }
     }
 
     private loadData = (area: string) => {
-        const [{}, actions] = this.context;
-        actions.getObjectives({area: area})
+        const [{ }, actions] = this.context;
```
Comment (beccamc) [inline src/DetailView/DetailOKRList.tsx]: @Nianwens The objectives here are getting reset to only this area's objectives. I thought the objectives were always all of them. Can I change this? What is your expectation?

File: src/Resources/ToDo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 96 96" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Icons_Checklist_LTR" overflow="hidden"><g><path d="M23 14 73 14 73 82 23 82 23 14ZM17 88 79 88 79 8 17 8 17 88Z"/><rect x="50" y="23" width="17" height="4"/><rect x="50" y="39" width="17" height="4"/><rect x="50" y="71" width="17" height="4"/><rect x="50" y="55" width="17" height="4"/><path d="M43.8 20.8 41 18 34.4 24.6 31.8 22 29 24.8 34.4 30.2Z"/><path d="M43.8 36.8 41 34 34.4 40.6 31.8 38 29 40.8 34.4 46.2Z"/><path d="M43.8 52.8 41 50 34.4 56.6 31.8 54 29 56.8 34.4 62.2Z"/><path d="M43.8 68.6 41 65.8 34.4 72.4 31.8 69.8 29 72.6 34.4 78Z"/></g></svg>
```
Comment (edwindotcom) [inline src/Resources/ToDo.svg]: I'm getting an error when I try to deploy this: 

```Error processing 'edwon.okr-hub-dev' SVG file types in extension package are not supported. File at '/dist/fonts/ToDo.svg' is of type SVG, please convert it to a different format (like PNG or JPG) and try again.
```

Comment (beccamc) [reply src/Resources/ToDo.svg]: This is happening when you try to upload to a cloud instance?

Comment (edwindotcom) [reply src/Resources/ToDo.svg]: it happens when I deploy this branch to my personal dev/cloud instance

Comment (edwindotcom) [reply src/Resources/ToDo.svg]: and i get this error in my extension install log at `https://dev.azure.com/edwon/...`

Comment (beccamc) [reply src/Resources/ToDo.svg]: Okay, I'm looking into it

**Pull request merged**